pytest_timeout.py

```diff
@@ -39,58 +38,61 @@
-                    type=float,
-                    help=TIMEOUT_DESC)
-    group.addoption('--timeout_method',
-                    action='store',
-                    choices=['signal', 'thread'],
-                    help='Deprecated, use --timeout-method')
-    group.addoption('--timeout-method',
-                    dest='timeout_method',
-                    action='store',
-                    choices=['signal', 'thread'],
-                    help=METHOD_DESC)
-    parser.addini('timeout', TIMEOUT_DESC)
-    parser.addini('timeout_method', METHOD_DESC)
-    parser.addini('timeout_func_only', FUNC_ONLY_DESC, type='bool')
+        "timeout",
+        "Interrupt test run and dump stacks of all threads after " "a test times out",
+    )
+    group.addoption("--timeout", type=float, help=TIMEOUT_DESC)
+    group.addoption(
+        "--timeout_method",
+        action="store",
+        choices=["signal", "thread"],
+        help="Deprecated, use --timeout-method",
+    )
+    group.addoption(
+        "--timeout-method",
+        dest="timeout_method",
+        action="store",
+        choices=["signal", "thread"],
+        help=METHOD_DESC,
+    )
+    parser.addini("timeout", TIMEOUT_DESC)
+    parser.addini("timeout_method", METHOD_DESC)
+    parser.addini("timeout_func_only", FUNC_ONLY_DESC, type="bool")
 
 
 @pytest.hookimpl
 def pytest_configure(config):
     # Register the marker so it shows up in --markers output.
     config.addinivalue_line(
-        'markers',
-        'timeout(timeout, method=None, func_only=False): Set a timeout, timeout '
-        'method and func_only evaluation on just one test item.  The first '
-        'argument, *timeout*, is the timeout in seconds while the keyword, '
-        '*method*, takes the same values as the --timeout_method option. The '
-        '*func_only* keyword, when set to True, defers the timeout evaluation '
-        'to only the test function body, ignoring the time it takes when '
-        'evaluating any fixrures used in the test.')
+        "markers",
+        "timeout(timeout, method=None, func_only=False): Set a timeout, timeout "
+        "method and func_only evaluation on just one test item.  The first "
+        "argument, *timeout*, is the timeout in seconds while the keyword, "
+        "*method*, takes the same values as the --timeout_method option. The "
+        "*func_only* keyword, when set to True, defers the timeout evaluation "
+        "to only the test function body, ignoring the time it takes when "
+        "evaluating any fixrures used in the test.",
+    )
 
     settings = get_env_settings(config)
     config._env_timeout = settings.timeout
     config._env_timeout_method = settings.method
     config._env_timeout_func_only = settings.func_only
 
 
-
 @pytest.hookimpl(hookwrapper=True)
 def pytest_runtest_protocol(item):
     func_only = get_func_only_setting(item)
     if func_only is False:
         timeout_setup(item)
-    outcome = yield
+    yield
```

Comment:
Is this black doing this?  Does it respect a rustfmt-like convention of using `_outcome` for unused variables?  I do rather like having that variable assignment as it gives a lot of clues to the reader.

Reply:
This wasn't black, it was flake8

Reply:
sure, not worth having an argument over.  if the flake8 gods have decided so be it